frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -152,12 +148,12 @@ const PropertyListView: React.FC = () => {
       // Only update the data if this is the latest fetch
       if (fetchId === fetchIdRef.current && agencyIds?.length > 0) {
         const query = getServerQuery({ pageIndex, pageSize, filter, agencyIds });
-        const response = await CustomAxios().get<IPagedItems<IProperty>>(getPropertyListUrl(query));
+        const data = await service.getPropertyList(query);
```

Comment:
Use a `service` class/helper instead of axios. In my opinion, this level of indirection allows for easier mocking in tests.